packages/yourloops/lib/auth/api.ts

```diff
@@ -268,13 +345,80 @@ async function requestPasswordReset(
   }
 }
 
-async function resetPassword(key: string | null, username: string, password: string, traceToken: string): Promise<boolean> {
+async function sendAccountValidation(auth: Readonly<Session>, traceToken: string, language = "en"): Promise<boolean> {
```

Comment:
The trace token is already in the session, you do not need to pass it down again